ajd/My-List-Util/t/hosts.t

```diff
@@ -0,0 +1,28 @@
+#!perl -T
+use 5.006;
+use strict;
+use warnings;
+use Test::More;
+use Test::File;
+
+plan tests => 2;
+
+diag( "Testing your hosts file, Perl $], $^X" );
+
+SKIP: {
+  if ( $^O ne 'linux' ) {
+    skip ( 'This is not linux', 1 );
+  }
+
+  file_exists_ok( '/etc/hosts/' );
+
+}
+
+SKIP: {
+  if ( $^O ne 'windows' ) {
```

Comment:
Apparently it never will be, Windows is down as `MSWin32` (see [perlvar#$^O](http://perldoc.perl.org/perlvar.html#%24OSNAME)).